plugins/CoreHome/templates/_topBar.twig

```diff
@@ -1,4 +1,4 @@
-{{ postEvent("Template.beforeTopBar", userAlias, userLogin, topMenu) }}
+{{ postEvent("Template.beforeTopBar", userLogin, topMenu) }}
```

Comment:
Should we mention this change in the changelog?

Reply:
@diosmosis because they are defined in twig itself, they are not official events anyway so shouldn't be needed (see https://developer.matomo.org/api-reference/events where they aren't listed). Also checked in our code we don't seem to use them.